package.json

```diff
@@ -11,8 +11,8 @@
     "@storybook/storybook-deployer": "^2.0.0",
     "axios-mock-adapter": "^1.10.0",
     "babel-cli": "^6.10.1",
-    "babel-core": "^6.26.3",
-    "babel-eslint": "^6.1.2",
+    "babel-core": "^6.26.0",
+    "babel-eslint": "^8.2.3",
```

Comment:
@sharvit can you create packaging PR's to match? thanks. (or remove unneeded changes?)

Reply:
@ohadlevy are you sure we need a packaging PR here?
I can't find `babel-eslint` in our current packaging.